app/Models/Photo.php

```diff
@@ -22,4 +22,12 @@ public function post()
     {
         return $this->morphOne('Rogue\Models\Post', 'postable');
     }
+
+    /**
+     * Get all of the photo's reactions.
+     */
+    public function reactions()
+    {
+        return $this->morphMany('Rogue\Models\Reaction', 'reactionable');
```

Comment:
Total nitpick, but let's use `::class` notation when referencing classes. That way we can catch any silent errors that might otherwise occur from typos. It also saves you a few keystrokes! 😄

```diff
-        return $this->morphMany('Rogue\Models\Reaction', 'reactionable');
+        return $this->morphMany(Reaction::class, 'reactionable');
```